store/mongo/migration_2_0_0.go

```diff
@@ -148,6 +139,7 @@ func (m *migration_2_0_0) Up(from migrate.Version) error {
 				writes = append(writes, mongo.NewInsertOneModel().SetDocument(item))
 			}
 			if len(writes) == findBatchSize {
+				_, err = collOut.BulkWrite(ctx, writes)
```

Comment:
oh wow, what a shame. I actually for got to call Write, the error handling is there, I simply did not put the Write here. I am too ashamed and not qualified to review. looks good.

Reply:
I'm more surprised that it passed the testing infrastructure. I think there's some room for improvement when testing database migrations.
In any case, I'd appreciate a review from you, if you have the time.